infra/gcp/namespaces/namespace-user-role.yml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: {{namespace}}
 rules:
   - apiGroups: [""]
-    resources: ["configmaps", "endpoints", "persistentvolumeclaims", "pods", "resourcequotas", "services"]
+    resources: ["configmaps", "endpoints", "persistentvolumeclaims", "pods", "pods/log", "pods/status", "resourcequotas", "services"]
```

Comment:
I don't think we need pods/status.  Users can read the status through 'pods'.  'pods/status' is only needed if you want to write status, which we don't really need.

We may need pods/exec ?